client/src/components/core/ms-dropdown/MsDropdownPopover.vue

```diff
@@ -29,22 +29,18 @@
         :class="{ selected: selectedOption?.key === option.key }"
         v-if="selectedOption?.key === option.key"
       />
-      <ion-icon
+      <ms-information-tooltip-icon
         v-if="option.disabled && option.disabledReason"
-        slot="end"
```

Comment:
slot should still be passed since this is specific to the dropdown popover case.